src/index.js

```diff
@@ -5,19 +5,32 @@ import { createTouchPopup } from './touchPopup'
 import { renderPreview, renderLoading, renderError, renderDisambiguation, renderOffline } from './preview'
 import { getWikipediaAttrFromUrl, isTouch, getDir, isOnline } from './utils'
 
+const invokeCallback = ( events, name, ...params ) => {
```

Comment:
```suggestion
const invokeCallback = ( events, name, params ) => {
```

this should be just `params` as it is already an array